.github/workflows/perf.yml

```diff
@@ -43,4 +45,9 @@ jobs:
       - uses: ./github-actions/npm/checkout-and-setup-node
       - uses: ./github-actions/bazel/setup
       - run: yarn install --immutable
-      - run: yarn ng-dev perf workflows --name ${{ matrix.workflow }}
+      - uses: 'google-github-actions/auth@v2'
```

Comment:
Do you mind adding some information on the workload identity provider setup for future help?